<Document accessor="public" category="Function" name="SendFailure" declFile="WfLibraryPredefined.h" declId="FB$vl::reflection::description::IPromise::SendFailure@bool(::vl::Ptr&lt;::vl::reflection::description::IValueException&gt;)[decl0]"><summary>Mark the <symbol name="IFuture" docId="vl::reflection::description::IFuture" declFile="WfLibraryPredefined.h" declId="NI$vl::reflection::description::IFuture"/> object as finished by providing an exception.</summary><returns>Returns true when this operation succeeded. Multiple calls to <symbol name="SendResult" docId="vl::reflection::description::IPromise::SendResult@bool(::vl::reflection::description::Value const &amp;)" declFile="WfLibraryPredefined.h" declId="FB$vl::reflection::description::IPromise::SendResult@bool(::vl::reflection::description::Value const &amp;)[decl0]"/> and <symbol name="SendFailure" docId="vl::reflection::description::IPromise::SendFailure@bool(::vl::Ptr&lt;::vl::reflection::description::IValueException&gt;)" declFile="WfLibraryPredefined.h" declId="FB$vl::reflection::description::IPromise::SendFailure@bool(::vl::Ptr&lt;::vl::reflection::description::IValueException&gt;)[decl0]"/> cause a failure.</returns><param name="failure">The exception of the <symbol name="IFuture" docId="vl::reflection::description::IFuture" declFile="WfLibraryPredefined.h" declId="NI$vl::reflection::description::IFuture"/> object.</param><signature><![CDATA[bool SendFailure(
    Ptr<IValueException> failure
);
]]></signature><seealsos><symbol name="Ptr" docId="vl::Ptr" declFile="Pointer.h" declId="NI$vl::Ptr"/><symbol name="IValueException" declFile="GuiTypeDescriptorPredefined.h" declId="NI$vl::reflection::description::IValueException"/></seealsos></Document>